public/static/locales/en/manageProjects.json

```diff
@@ -218,10 +218,8 @@
     "landOwnershipType": "Land Ownership Type",
     "restorationProject": "Restoration Project",
     "addProjetDescription": "You're few steps away from showcasing your work on the Plant-for-the-Planet platform.",
-    "englishOnly": " Please create projects in English only.",
-    "important": "Important:",
-    "addProjetContact": "If you need any help contact us at ",
-    "supportLink": "support@plant-for-the-planet.org",
+    "createProjectsEnglishOnly": "<importantContainer>Important:</importantContainer>Please create projects in English only.",
```

Comment:
I'd suggest using `noteLabel` instead of `importantContainer` - I'd use `important` for text that is important to emphasize i.e. a key bit of information. In this case, the text "Important" is just a label, and not the information that needs highlighting.